apps/vr-tests-react-components/src/stories/Field.stories.tsx

```diff
@@ -109,6 +109,44 @@ storiesOf('Field', module)
       <Checkbox label="Checkbox in a horizontal field" />
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 7 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27032&commitId=af8e089011664c9f7cf1f72e96717024938e9ba7&env=prod)

✅ There was 7 screenshots added, 0 screenshots removed, 1862 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 7 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Field.infoButton+horizontal.default.chromium.png|0| Added|
|Field.infoButton+longLabel.default.chromium.png|0| Added|
|Field.infoButton+noLabel.default.chromium.png|0| Added|
|Field.infoButton+required.default.chromium.png|0| Added|
|Field.infoButton+size-large.default.chromium.png|0| Added|
|Field.infoButton+size-small.default.chromium.png|0| Added|
|Field.infoButton.default.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv9"/>